repair/repair.cc

```diff
@@ -439,8 +439,7 @@ void repair_module::abort_all_repairs() {
         auto it = _tasks.find(x.second);
         if (it != _tasks.end()) {
             auto& impl = dynamic_cast<shard_repair_task_impl&>(*it->second->_impl);
-            // FIXME: use generic repair task abort method
-            impl.abort_repair_info();
+            (void)impl.abort();
```

Comment:
Why convert to `void`? If this is to oppress the error about the discarded future, please add a comment explaining who waits for this fiber.